pandas/tseries/resample.py

```diff
@@ -14,7 +21,560 @@
 import pandas.tslib as tslib
 
 
-_DEFAULT_METHOD = 'mean'
+class Resampler(_GroupBy):
+
+    # to the groupby descriptor
+    _attributes = ['freq', 'axis', 'closed', 'label', 'convention',
+                   'loffset', 'base', 'kind']
+
+    # API compat of allowed attributes
+    _deprecated_valids = _attributes + ['_ipython_display_', '__doc__',
+                                        '_cache', '_attributes', 'binner',
+                                        'grouper', 'groupby', 'keys',
+                                        'sort', 'kind', 'squeeze',
+                                        'group_keys', 'as_index',
+                                        'exclusions']
+
+    # API compat of disallowed attributes
+    _deprecated_invalids = ['iloc', 'loc', 'ix']
```

Comment:
putting these here means that only getting the values (not setting) raises an error instead of raising a warning and giving you the back compat return value. I suppose that is on purpose? 
That means that the explanation in the whatsnew is not fully complete as there it only says that 'assignments' will raise an error.